scripts/generate_default_results.py

```diff
@@ -13,15 +12,18 @@
 
 
 def generate_default_results():
-    # Keeping computations short with option to keep file under 1000 FF
-    # DUMMY_DATA = DUMMY_DATA[(DUMMY_DATA["idmen"] > 2500) & (DUMMY_DATA["idmen"] < 7500)]
-    bulk_data_simulation, data_by_entity = simulation(PERIOD, DUMMY_DATA, TBS)
     # precalcul cas de base sur la population pour le cache
-    base_results = data_by_entity["foyer_fiscal"][["wprm", "idfoy"]]
-    base_results["avant"] = bulk_data_simulation.calculate("irpp", PERIOD)
-    simulation_plf_deciles = simulation(PERIOD, DUMMY_DATA, TBS_PLF)
-    base_results["plf"] = simulation_plf_deciles[0].calculate("irpp", PERIOD)
-    base_results[["idfoy", "avant", "plf", "wprm"]].to_csv(
+    base_results = None
+    liste_base_reformes = []
+    for reforme in TBS_DEFAULT:
+        liste_base_reformes += [reforme]
+        bulk_data_simulation, data_by_entity = simulation(
+            PERIOD, DUMMY_DATA, TBS_DEFAULT[reforme]
+        )
+        if base_results is None:
+            base_results = data_by_entity["foyer_fiscal"][["wprm", "idfoy"]]
+        base_results[reforme] = bulk_data_simulation.calculate("irpp", PERIOD)
+    base_results[["idfoy"] + liste_base_reformes + ["wprm"]].to_csv(
```

Comment:
Ajouter une ligne pour la lisibilité ?
```suggestion

    base_results[["idfoy"] + liste_base_reformes + ["wprm"]].to_csv(
```